.github/workflows/genesis_files_for_relay.yml

```diff
@@ -17,53 +17,53 @@ jobs:
     steps:
       - name: get curl
         run: |
-          apt update
-          apt install -y curl
+          sudo apt update
+          sudo apt install -y curl
       - name: fetch and chmod manta release
         run: |
-          curl -L -o $HOME/manta https://github.com/Manta-Network/Manta/releases/download/$VERSION/manta
-          chmod +x $HOME/manta
-          ls -ahl $HOME
+          curl -L -o manta https://github.com/Manta-Network/Manta/releases/download/$VERSION/manta
+          chmod +x manta
+          ls -ahl .
       - name: Export genesis files
         run: |
-          $HOME/manta build-spec --chain $CHAIN --raw > ./$CHAIN-genesis.json
-          $HOME/manta export-genesis-state --chain $CHAIN > $CHAIN-genesis.state
-          $HOME/manta export-genesis-wasm --chain $CHAIN > $CHAIN-genesis.wasm
+          ./manta build-spec --chain $CHAIN --raw > $CHAIN-genesis.json
+          ./manta export-genesis-state --chain $CHAIN > $CHAIN-genesis.state
+          ./manta export-genesis-wasm --chain $CHAIN > $CHAIN-genesis.wasm
```

Comment:
I think parachain should be started after both files are generated, then we manually check the logs, compare `genesis.state`.
And maybe print the binary version as well.
```
./manta --version
```

Reply:
As a first step, let's do that manually after generating these or integrate it into publish-draft-releases.
this workflow is meant only to create the genesis files.